<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTextEdit" name="output_text"/>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QPlainTextEdit" name="input_text">
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="setting_btn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>setting</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="send_btn">
      <property name="text">
       <string>send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>489</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Setting">
    <property name="title">
     <string>&amp;Setting</string>
    </property>
   </widget>
   <addaction name="menu_Setting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>send_btn</sender>
   <signal>clicked()</signal>
   <receiver>input_text</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
